libraries/api/src/api.js

```diff
@@ -31,35 +30,35 @@ export default class API {
     assert(options.monitor, 'monitor option is required');
     assert(!options.referencePrefix, 'referencePrefix is now deprecated!');
 
-    options = _.defaults({}, options, {
+    options = Object.assign({}, {
```

Comment:
I think Object spread works here and may be a big simpler.

e.g. something like:

```js
options = {
  ...options,
  inputLimit: '10mb',
  allowedCORSOrigin: '*',
}
```

Also in TS, the type inference will work better if you assign it to a new name.

```js
const resolvedOptions = {
  ...options,
  inputLimit: '10mb',
  allowedCORSOrigin: '*',
}
```

Reply:
Thanks! Is new name works better because TS doesn't need to merge types of two different object, and just infer from the new one?